SpringProject/src/test/java/com/gradproject2019/UserConferenceServiceTest.java

```diff
@@ -48,4 +50,18 @@ public void shouldSaveInterestInConference() {
         assertThat(userConference.getUserId()).isEqualTo(userConferenceResponseDto.getUserId());
 
     }
+
+    @Test
+    public void shouldGetInterestInConference() {
+        UserConference userConference = anUserConference()
+                .withUserId(1L)
+                .withConferenceId(1L)
+                .build();
+        given(userConferenceRepository.saveAndFlush(any(UserConference.class))).willReturn(userConference);
+
+        List<ConferenceResponseDto> conferenceResponseDtoList = userConferenceService.getConferenceByUserId(token.getToken(), userConference.getUserId());
+
+        assertThat(userConference.getConferenceId()).isEqualTo(conferenceResponseDtoList.get(0).getId());
```

Comment:
Because you won't know the size of the list at this point, maybe you should assert that there's only one element here